.maintain/sentry-node/docker-compose.yml

```diff
@@ -80,9 +83,9 @@ services:
       - "--port"
       - "30333"
       - "--charlie"
-      - "--bootnodes"
+      - "--sentry"
       - "/dns4/validator-a/tcp/30333/p2p/QmRpheLN4JWdAnY7HGJfWFNbfkQCb6tFf4vvA6hgjMZKrR"
-      - "--bootnodes"
```

Comment:
@ddorgan sentry-a is a sentry node of validator a only, not validator b. Thus this would still need to be `--bootnode`. I will push another commit.